modules/safer-cluster/README.md

```diff
@@ -0,0 +1,15 @@
+# Safer Beta Cluster
+
+The module defines a safer configuration for a GKE cluster.
+
+TODO(mmontan): add documentation for the module.
```

Comment:
Please add your README from https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/281 into this PR